unit_tests/test_events.py

```diff
@@ -10,13 +10,15 @@
 
 from sdcm.prometheus import start_metrics_server
 from sdcm.sct_events import (start_events_device, stop_events_device, Event, TestKiller,
-                             InfoEvent, CassandraStressEvent, CoreDumpEvent, DatabaseLogEvent, DisruptionEvent, DbEventsFilter, SpotTerminationEvent,
+                             InfoEvent, CassandraStressEvent, ScyllaBenchEvent, CoreDumpEvent, DatabaseLogEvent, DisruptionEvent, DbEventsFilter, SpotTerminationEvent,
                              KillTestEvent, Severity, ThreadFailedEvent)
 
 LOGGER = logging.getLogger(__name__)
 
 logging.basicConfig(format="%(asctime)s - %(levelname)-8s - %(name)-10s: %(message)s", level=logging.DEBUG)
 
+# pylint: disable=too-many-public-methods
```

Comment:
why here, and not on the calls it's related to ?

Reply:
@fruch AFAIK it's per class. max-public-methods is 20 by default. May be we need to increase it?

Reply:
since this is a test, it's not that impotent, but my general comment here, that we should disable on a file level like that, this commend need to go in the same line of the class declaration it's related to. i.e. like that it's affecting the whole file, and other classes defined in this file.

if 20 is too low, maybe. but it's just a thumb rule, to help us think if a refactor is needed. that the whole point of those specific warnings. hitting them again and again and enlarging the number is like disabling them.

Reply:
up it